content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -48,7 +48,7 @@ but you may also on other OSes.
 | Config file API           | alpha          |
 | Self-hosting              | alpha          |
 | kubeadm alpha subcommands | alpha          |
-| CoreDNS                   | alpha          |
+| CoreDNS                   | GA             |
```

Comment:
this was changed in https://github.com/kubernetes/website/pull/8753/files

Reply:
Yeah. Did not find #8753 until I filed this.